source/configuration_manual/mail_location/obox/dictmap.rst

```diff
@@ -183,10 +183,12 @@ Dictmap Parameters
 |                                 | .. versionadded:: v2.3.2.1                                                   |
 +---------------------------------+------------------------------------------------------------------------------+
 | max-parallel-iter=<n>           | Describes how many parallel dict iterations can be created internally. The   |
-|                                 | default value is 1. Parallel iterations can especially help speed up reading |
-|                                 | huge folders.                                                                |
+|                                 | default value is 10. Parallel iterations can especially help speed up        |
+|                                 | reading huge folders.                                                        |
 |                                 |                                                                              |
 |                                 | .. versionadded:: v2.3.10                                                    |
+|                                 | .. versionchanged:: v3.0.0 increase default to 10 earlier versions run with  |
+|                                 |                     1 as default                                             |
```

Comment:
Needs a new sentence for Earlier versions.. Also the first sentence would look better as "Increased ..."

Reply:
Oh, and v2.4.0;v3.0.0 as the version